bigquery/google/cloud/bigquery_v2/proto/encryption_config_pb2.py

```diff
@@ -3,13 +3,11 @@
 # source: google/cloud/bigquery_v2/proto/encryption_config.proto
 
 import sys
-
-_b = sys.version_info[0] < 3 and (lambda x: x) or (lambda x: x.encode("latin1"))
+_b=sys.version_info[0]<3 and (lambda x:x) or (lambda x:x.encode('latin1'))
```

Comment:
@busunkim96 What's going on with this PR? Does synthtool not run the blacken session anymore?

Reply:
The Autosynth image only has 3.6, so the blacken session was skipped. I made a change to the noxfile